core/src/components/modal/animations/sheet.ts

```diff
@@ -12,7 +12,13 @@ export const createSheetEnterAnimation = (opts: ModalAnimationOptions) => {
    * current breakpoint, then the backdrop should be fading in.
    */
   const shouldShowBackdrop = backdropBreakpoint === undefined || backdropBreakpoint < currentBreakpoint!;
-  const initialBackdrop = shouldShowBackdrop ? `calc(var(--backdrop-opacity) * ${currentBreakpoint!})` : '0';
+
+  const initialBackdrop =
+    opts.theme === 'ionic'
+      ? staticBackdropOpacity
+      : shouldShowBackdrop
+      ? `calc(var(--backdrop-opacity) * ${currentBreakpoint!})`
+      : '0';
```

Comment:
Let's expand this to make it easier to read. Nested ternary can be hard to read.

Example:
```suggestion
  let initialBackdrop = '0';
  if (opts.theme === 'ionic') {
     initialBackdrop = staticBackdropOpacity;
  } else if (shouldShowBackdrop) {
     initialBackdrop = `calc(var(--backdrop-opacity) * ${currentBreakpoint!})`;
  }
```